config/parse.go

```diff
@@ -308,6 +308,9 @@ func resolveHandlers(c *Config) error {
 		if h.Parser == "protobuf" {
 			logger.Debug("Using protobuf parser")
 			h.Handler.Parser = parser.ProtoBuf{}
+		} else if h.Parser == "custom-json" {
```

Comment:
Yeah.... It's probably just an other type of module. Then we can just check parser.Auto["custom-json"] or similar. We don't really need to expose the module-aspect in configuration of course, since there thus far isn't any configuration for any of the parsers. But I'm fine with this for now. It's not exactly super-important yet. But I _do_ predict more parsers in the future.